apps/scout-agents/src/__tests__/healthCheck.spec.ts

```diff
@@ -0,0 +1,23 @@
+import type { Server } from 'http';
+
+import request from 'supertest';
+
+import { app } from '../server';
+
+let server: Server;
+
+beforeAll(() => {
+  server = app.listen();
+});
+
+afterAll((done) => {
+  server.close(done);
+});
+
+describe('test', () => {
+  test('should return 200', async () => {
+    // const response = await request(server).get(`/api/health`);
+    // expect(response.status).toBe(200);
+    // TODO: Fix local test runner
```

Comment:
remember to clean this up or just delete the test ,,

Reply:
I left this as a reminder as I'm having issues with jest. Keeps giving me an error with the local runner

I definitely want to have at least 1 working test